devops/scripts/benchmarks/README.md

```diff
@@ -9,47 +9,68 @@ Scripts for running performance tests on SYCL and Unified Runtime.
 - [LlamaCpp Benchmarks](https://github.com/ggerganov/llama.cpp)
 - [SYCL-Bench](https://github.com/unisa-hpc/sycl-bench)
 
-## Running
+## Requirements
 
-`$ ./main.py ~/benchmarks_workdir/ --sycl ~/llvm/build/ --ur ~/ur --adapter adapter_name`
+Built compiler to be used for benchmarks.
+Instructions on where to find releases or how to build from sources can be found [here](https://github.com/intel/llvm).
 
-This will download and build everything in `~/benchmarks_workdir/` using the compiler in `~/llvm/build/`, UR source from `~/ur` and then run the benchmarks for `adapter_name` adapter. The results will be stored in `benchmark_results.md`.
+[Unified Runtime](https://github.com/intel/llvm/tree/sycl/unified-runtime) installed.
+Path to the UR install directory will be required in case of using UR for benchmarking.
 
-The scripts will try to reuse the files stored in `~/benchmarks_workdir/`, but the benchmarks will be rebuilt every time. To avoid that, use `--no-rebuild` option.
+`Python3` is required to install and run benchmarks.
 
-## Running in CI
+## Building & Running
 
-The benchmarks scripts are used in a GitHub Actions worflow, and can be automatically executed on a preconfigured system against any Pull Request.
+```bash
+$ git clone https://github.com/intel/llvm.git
+$ cd llvm/devops/scripts/benchmarks/
+$ pip install -r requirements.txt
 
-![compute benchmarks](workflow.png "Compute Benchmarks CI job")
+$ ./main.py ~/benchmarks_workdir/ --sycl ~/llvm/build/ --ur ~/ur --adapter adapter_name
```

Comment:
I think we should mention that `--ur` needs to point to UR installation, not just a build. And to do that, we need to use `-DCMAKE_INSTALL_PREFIX=~/ur_install` so it's not installed globally.
(maybe a good idea to show sycl and ur compilation with both adapters?)

Reply:
I added information about the installation folder and a UR `cmake` tip. However, I'm not convinced about the SYCL compilation guide in this README.